app/views/providers/proceeding_merits_task/prohibited_steps/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= form_with(
+      model: @form,
+      url: providers_merits_task_list_prohibited_steps_path(@proceeding),
+      method: :patch,
+      local: true
+    ) do |form| %>
+  <%= page_template(page_title: t('.h1-heading'),
+                    template: :basic,
+                    back_link: {},
+                    form: form) do %>
+    <%= form.govuk_radio_buttons_fieldset :uk_removal, legend: {size: 'xl', tag: 'h1', text: page_title} do %>
+      <%= form.govuk_radio_button :uk_removal, true, link_errors: true, label: {text: t('generic.yes')} %>
+      <%= form.govuk_radio_button :uk_removal, false, label: {text: t('generic.no')} do %>
+        <%= form.govuk_text_area :details, label: {text: t('.no-hint')}, rows: 5 %>
+      <% end %>
+
+    <% end %>
+    <div class="govuk-!-padding-bottom-4"></div>
```

Comment:
It would be much cleaner to add the spacing class name `govuk-!-padding-bottom-4` to `govuk_radio_buttons_fieldset` and remove this empty element, that only purpose is to apply spacing.